templates/generic/web-app/tests/vitest.config.ts.hbs

```diff
@@ -2,7 +2,10 @@ import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
   test: {
-    threads: false,
-    testTimeout: 60*1000*4 // 4  mins
+    retry: 3,
+    testTimeout: 60*1000*6, // 4  mins
+    poolOptions: {
+      threads: { singleThread: true },
```

Comment:
Agree with this, have configured some of my tests this way, and the same with sweettest. Running multiple tests at the same time that run Holochain is slow and unpredictable